public/components/agents/vuls/inventory/detail.tsx

```diff
@@ -272,6 +307,32 @@ export class Details extends Component {
     );
   }
 
+  renderExternalReferences(references) {
+    return (
+      <EuiAccordion
+        id={Math.random().toString()}
```

Comment:
**nitpick**: You could use a long string defining the part of UI. For example: `modules_vulnerabilities_inventory_flyout_details_references`.